lib/do_auth/web/confirm.ex

```diff
@@ -10,8 +10,10 @@ defmodule DoAuth.Web.Confirm do
   alias Uptight.Text, as: T
   alias Uptight.Result
   alias DoAuth.User
+  alias DoAuth.Credential
 
-  plug(:confirm)
+  plug :confirm
+  plug :render_result
```

Comment:
The only question (which could be fun to mention in the documentation) is how to make a Router that *just* calls `confirm` plug and then has a custom `render_result`.

I know it's probably mentioned somewhere in Plug docs, but it would be just a good thing to know for the users, since our default way of calling Plugs is module-based and will call both in succession (as far as I understand and based on what I see in tests).